src/praisonai/praisonai/ui/chat.py

```
@@ -828,7 +900,7 @@ async def on_chat_resume(thread: ThreadDict):
     cl.user_session.set("message_history", message_history)
     
     # Pre-create agent for faster first response
-    _get_or_create_agent(model_name, tools_enabled)
+    _get_or_create_agent(model_name, tools_enabled, selected_external_agents)
```

Ruff (0.15.10)

[error] Undefined name `selected_external_agents`

(F821)